rosdep/python.yaml

```
@@ -181,6 +181,16 @@ ds4drv-pip:
   ubuntu:
     pip:
       packages: [ds4drv]
+emcee-pip:
+  debian:
+    pip:
+      packages: [emcee]
+  fedora:
+    pip:
+      packages: [emcee]
+  ubuntu:
+    pip:
+      packages: [emcee]
```

This can be simplified using an os-level wildcard.

```suggestion
  '*':
    pip:
      packages: [emcee]
```